### PR TITLE
CIRC-381 primitives and constants

### PR DESCRIPTION
**Pull request by marcjohnson-kint**
## Purpose
Ongoing series of pull requests to reduce the noise created by many outstanding sonar warnings

## Approach
* Address warnings relating to constants and primitive types

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/java/org/folio/circulation/domain/ServicePoint.java [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
-  
+
   public ServicePoint(JsonObject representation) {
     this.representation = representation;
   }
-  
-  public Boolean isPickupLocation() {
+
+  public boolean isPickupLocation() {
     return representation.getBoolean("pickupLocation");
```
Comment (bohdan-suprun) [inline src/main/java/org/folio/circulation/domain/ServicePoint.java:18]: Should we use default value here? It is possible to get NPE here, when the property is undefined.
```suggestion
    return representation.getBoolean("pickupLocation", false);
```

File: src/main/java/org/folio/circulation/domain/User.java [hunk 2]
```diff
@@ -35,17 +35,17 @@ public boolean canDetermineStatus() {
...
 
-  public Boolean isInactive() {
+  public boolean isInactive() {
     return !isActive();
   }
 
-  Boolean isActive() {
-    final Boolean active = representation.getBoolean("active");
+  boolean isActive() {
+    final boolean active = representation.getBoolean("active");
```
Comment (bohdan-suprun) [inline src/main/java/org/folio/circulation/domain/User.java:43]: The same as for previous, is it required property, if no then we need a default value for it? 
```suggestion
    final boolean active = representation.getBoolean("active", false);
```

**Comment by marcjohnson-kint** [comment]
@bohdan-suprun Apologies, I was on autopilot for merging. I will raise a new pull request that addresses your feedback. Apologies :-/

**Comment by bohdan-suprun** [comment]
> @bohdan-suprun Apologies, I was on autopilot for merging. I will raise a new pull request that addresses your feedback. Apologies :-/

No prob

**Pull request merged**